samples/client/petstore/go/go-petstore/api/openapi.yaml

```diff
@@ -1182,15 +1182,12 @@ components:
       example:
         photoUrls:
         - photoUrls
-        - photoUrls
```

Comment:
I don't understand these changes. The duplicated list items are a little strange, but why do we need to remove them for this change?